src/com/gfk/jenkins/components/Configuration.groovy

```diff
@@ -0,0 +1,17 @@
+package com.gfk.jenkins.components
+
+class Configuration {
+    def steps
+    Configuration(steps) {this.steps = steps}
+    // branches that jenkins will push
+    public static DEPLOY_BRANCH_PATTERN_REGEX_MATCH = '^(master|develop|release/.*|feature/.*|hotfix/.*|PR-.*)$'
+    public static AWS_ACCOUNT = "123456789123"
```

Comment:
Can be a good idea set the variables AWS_ACCOUNT and AWS_REGION as jenkins credentials to be completely environment agnostic.
Also can be a good idea add the posibility to modify some of this variables on the Jenkinsfile because maybe different jobs need to use different AWS accounts or region.